### DISCUSSION TITLE
https://akulsrivastava.com/dev-tips/vscode/evaluate-mathematical-expressions-in-vscode

### DISCUSSION THREAD

**Discussion by giscus** [akulsr0/website#35] [Content]
# https://akulsrivastava.com/dev-tips/vscode/evaluate-mathematical-expressions-in-vscode

Evaluate mathematical expression in VSCode

https://akulsrivastava.com/dev-tips/vscode/evaluate-mathematical-expressions-in-vscode

**Comment by avking007** [comment]
test comment

**Reply by akulsr0** [reply]
working!!!
